src/lib/tls/tls_algos.h

```diff
@@ -8,6 +8,7 @@
 #define BOTAN_TLS_ALGO_IDS_H_
 
 #include <botan/types.h>
+#include <botan/asn1_obj.h>
```

Comment:
It will be, with TLS 1.3, because we introduced:

```C++
AlgorithmIdentifier algorithm_identifier_for_scheme(Signature_Scheme scheme);
```

and `AlgorithmIdentifier` is defined in this header. Removing for now.